packages/cell-toolbar-extension/src/index.ts

```diff
@@ -29,7 +31,24 @@ const cellToolbar: JupyterFrontEndPlugin<void> = {
     toolbarRegistry: IToolbarWidgetRegistry | null,
     translator: ITranslator | null
   ) => {
-    const toolbarItems =
+    let showCellToolbar: boolean | null;
+
+    /**
+     * Load the settings for this extension
+     *
+     * @param setting Extension settings
+     */
+    function loadSetting(setting: ISettingRegistry.ISettings | null): void {
+      // Read the setting and convert to the correct type
+      showCellToolbar =
+        setting === null
+          ? true
+          : (setting.get('showToolbar').composite as boolean);
+
+      extension.visible = showCellToolbar;
+    }
+
+    let toolbarItems =
```

Comment:
I do not see this change

```suggestion
    const toolbarItems =
```